.cargo/config.toml

```diff
@@ -1,6 +1,15 @@
 [build]
 
 [env]
+# Stack backtraces.
+RUST_BACKTRACE = "1"
+# Set output levels for `tracing` logging.
+RUST_LOG = "debug,dfir_rs=trace"
```

Comment:
Will this result in verbose logging for the DFIR runtime? Might not want that for the `trybuild` builds?

Reply:
Let's see

Reply:
Most error outputs are only printed when the tests fails (as `--nocapture` is NOT set)

Reply:
I feel like the hydro deploy trybuild builds don't actually obey the RUST_BACKTRACE/RUST_LOG set in the workspace or in `.cargo/config.toml`, which actually was an annoyance while trying to debug things a while back

Reply:
Oh really? Our trybuild logic copies over `.cargo/config.toml` which is why I was concerned with this. Like what happens if we do `cargo run -p hydro_test --example paxos`?